src/ci_providers/provider_drone.ts

```diff
@@ -14,7 +14,7 @@ function _getBuild(inputs: UploaderInputs): string {
 
 function _getBuildURL(inputs: UploaderInputs): string {
   const { environment: envs } = inputs
-  return envs.DRONE_BUILD_URL || ''
+  return envs.DRONE_BUILD_LINK || envs.DRONE_BUILD_URL || envs.CI_BUILD_URL || ''
```

Comment:
I can't find any documentation for `CI_BUILD_URL`. Is it documented somewhere?

Reply:
Found that in other sources collecting coverage for drone, not sure anymore where
Google turns up this e.g. https://codeclimate.com/github/codeclimate/ruby-test-reporter/lib/code_climate/test_reporter/ci.rb/source
It seems like a generic variable, so maybe we can keep that as a fallback?

Reply:
This appears to no longer(?) be a valid DroneCI env https://github.com/drone/runner-go/blob/f7a8ed14f07fb159568f28cd3e2841ce90417744/environ/environ.go

Reply:
Might be. But it seems it was and as it comes last it should be a better fallback than an empty string, isn't it?